stack_examples/aws_ssm_package_installer/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "region_resource" {
```

Comment:
You should add `description` fields here and `default` fields <if required>.

Reply:
agree with the description, but there is no default value to be set, is `default = ""` enough?